pandas/core/frame.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=E1101
+# pylint: disable=W0212,W0703,W0622
```

Comment:
i would rather not disable specific items for a whole file. Is there a reason for this?

Reply:
@jreback 100% agree, but these lines are already there in the original files, they've just been moved.

@alimcmaster1 removed some of the errors that were being ignored, but the ones left are not trivial.

Didn't check in detail, but `E1101` is probably to avoid false positives in attributes that are created dynamically and not found in the linting. It's disabled like this in around 80 files, so it may be worth to move it to `setup.cfg` and ignore it everywhere.

Didn't check the others, but I'd take care of those in a separate PR.

Reply:
ok that's fine.

Reply:
I'm merging #22863 as soon as the CI is green, so you may want to wait to remove E741, as it'll conflict.

Reply:
Merged. E741 should be removed from `setup.cfg` and `.pep8speaks.yml`. But not sure if the remaining tests are few. ;)

```
./pandas/tests/test_algos.py:375:9: E741 ambiguous variable name 'l'
./pandas/tests/test_multilevel.py:2065:9: E741 ambiguous variable name 'l'
./pandas/tests/frame/test_operators.py:796:9: E741 ambiguous variable name 'l'
./pandas/tests/frame/test_indexing.py:2079:9: E741 ambiguous variable name 'l'
./pandas/tests/frame/test_constructors.py:899:9: E741 ambiguous variable name 'l'
./pandas/tests/frame/test_constructors.py:926:9: E741 ambiguous variable name 'l'
./pandas/tests/frame/test_constructors.py:1747:9: E741 ambiguous variable name 'l'
./pandas/tests/frame/test_constructors.py:1752:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/common.py:260:24: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_frame.py:302:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_frame.py:310:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:545:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:560:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:575:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:595:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:611:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:642:9: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:953:17: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:966:17: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:984:17: E741 ambiguous variable name 'l'
./pandas/tests/plotting/test_datetimelike.py:998:17: E741 ambiguous variable name 'l'
./pandas/tests/indexing/test_loc.py:671:13: E741 ambiguous variable name 'l'
./pandas/tests/indexing/test_indexing.py:772:19: E741 ambiguous variable name 'l'
./pandas/tests/io/test_packers.py:515:9: E741 ambiguous variable name 'l'
./pandas/tests/io/test_packers.py:521:9: E741 ambiguous variable name 'l'
./pandas/tests/io/test_packers.py:529:9: E741 ambiguous variable name 'l'
./pandas/tests/io/test_pytables.py:2185:13: E741 ambiguous variable name 'l'
./pandas/tests/series/test_analytics.py:542:13: E741 ambiguous variable name 'l'
./pandas/tests/series/test_analytics.py:977:13: E741 ambiguous variable name 'l'
./pandas/tests/series/test_dtypes.py:246:9: E741 ambiguous variable name 'l'
./pandas/tests/series/test_dtypes.py:252:9: E741 ambiguous variable name 'l'
./pandas/tests/series/test_dtypes.py:273:9: E741 ambiguous variable name 'l'

```